### ISSUE TITLE
with_transporter_logs exposes elasticsearch password

### ISSUE THREAD

**Issue by leandropadua** [uken/fluent-plugin-elasticsearch#606]
#### Problem

Enabling `log_level = debug` and `with_transporter_log = true` exposes elasticsearch password when any failure happens on the connection to elasticsearch.

```
2019-07-16 23:18:03 +0000 [error]: #0 [out_es] [Faraday::ConnectionFailed] getaddrinfo: Name or service not known (SocketError) {:host=>"elasticsearch.es", :port=>9200, :scheme=>"http", :user=>"elastic", :password=>"changeme", :protocol=>"http"}
```

Is there a way to obfuscate this password?

#### Steps to replicate

Enable
```
with_transporter_log true
@log_level debug
```
and then force an error, like giving an wrong DNS name for elasticsearch url.

#### Expected Behavior or What you need to ask

Password to be obfuscated

#### Using Fluentd and ES plugin versions

I'm using this official image: https://github.com/fluent/fluentd-kubernetes-daemonset/tree/master/docker-image/v1.2/debian-elasticsearch

* OS version: `Debian GNU/Linux 9 (stretch)`
* Bare Metal or within Docker or Kubernetes or others? kubernetes
* Fluentd 
```
$ fluentd --version
fluentd 1.2.6
```
* ES plugin 3.x.y/2.x.y or 1.x.y
https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/docker-image/v1.2/debian-elasticsearch/Gemfile.lock
* ES version `6.5.0`

**Comment by cosmo0920** [comment]
This issue is caused by elasticsearch ruby client.
Could you report it in [its bug tracker](https://github.com/elastic/elasticsearch-ruby/issues)?

**Comment by leandropadua** [comment]
Thanks for chasing that up @cosmo0920 
Looking into their bug tracker, I found that they already reported it there: https://github.com/elastic/elasticsearch-ruby/issues/495
Apparently there's a fix merged as well, as per this PR => https://github.com/elastic/elasticsearch-ruby/pull/577

**Comment by leandropadua** [comment]
The image I'm using is referencing `elasticsearch-ruby` [v6.1.0](https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/docker-image/v1.2/debian-elasticsearch/Gemfile.lock#L16)

They have released the fix for `v6.2.0` => https://github.com/elastic/elasticsearch-ruby/releases/tag/6.2.0 => `Redacted password if host info is printed in error message`

Thanks a lot for pointing to the right direction @cosmo0920
